src/code/modules/remediation.js

```diff
@@ -0,0 +1,273 @@
+export const GUIDE_REMEDIATION_REQUESTED = "Guide remediation requested";
+export const STARTED_REMEDIATION = "Started remediation";
+export const ENDED_REMEDIATION = "Ended remediation";
+
+import { GUIDE_MESSAGE_RECEIVED, GUIDE_ALERT_RECEIVED } from './notifications';
+import { showNotification } from '../actions';
+import actionTypes from '../action-types';
+import GeneticsUtils from '../utilities/genetics-utils';
+
+const initialState = false;
+
+export default function notifications(state = initialState, action) {
+  switch (action.type) {
+    case GUIDE_REMEDIATION_REQUESTED:
+      if (action.attribute && action.challengeType) {
+        return {
+          attribute: action.attribute,
+          practiceCriteria: action.practiceCriteria,
```

Comment:
You're right, but at this point we would either need to change the ITS, or have a different name between the code and what the ITS is sending down. Neither option is great...

Actually, it's not clear that `practiceCriteria` will always be singular. It's basically a catch-all for "what other aspect besides the specific gene needs to be remediated?" Currently it's only used for `Sim`, and it's just one word, but it may end up being more expansive for the two challenge types we haven't gotten to yet (breeding and siblings).

It's a totally valid point, though, to have variable names that are as clear in their intent as possible.